packages/features/webhooks/lib/sendPayload.ts

```diff
@@ -46,6 +46,7 @@ function getZapierPayload(data: CalendarEvent & EventTypeInfo & { status?: strin
     description: data.description,
     customInputs: data.customInputs,
     responses: data.responses,
+    userFieldsResponses: data.userFieldsResponses,
```

Comment:
I pushed that change. We need to have the same payload in the zapier payload and in the /listBookings endpoint